src/Card.js

```diff
@@ -33,10 +35,13 @@ export default class Card {
 	renderToHTML(md, options){
 		// unify heading levels for consistent look in anki
 		for(let i = 0; i < this.front.length; i++)
-			if(this.front[i].type == 'heading_open' || this.front[i].type == 'heading_close') this.front[i].tag = 'h1';
+			if(this.front[i].type === 'heading_open' || this.front[i].type === 'heading_close') this.front[i].tag = 'h1';
 		// render front and back to html
 		let front = md.renderer.render(this.front, md.options, {});
 		let back = md.renderer.render(this.back, md.options, {});
+
+		back = `<style>${fs.readFileSync(`node_modules/@highlightjs/cdn-assets/styles/${options.codeStyle}.min.css`)}</style>` + back;
```

Comment:
Using the CDN assets is a great idea. However, loading them using a relative path like that won't work in any other directories than the repository root.

Reply:
Hmhm, I already suspected that this might be problematic. I'll see if I can find a more consistent way of linking there.

Reply:
Maybe we could use some hacky solution like [`es-dirname`](https://github.com/vdegenne/es-dirname)?

Reply:
As you're probably busy throughout the day, I will try implementing it myself

Reply:
Sounds good, feel free to directly push to this branch!